src/views/SignUp.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="login-body">
+    <div class="wrapper">
+      <div class="fullscreen">
+        <div class="fullscreen-body">
+          <signUpForm />
```

Comment:
You don't need to define separate component if you don't use it in several places and page doesn't have anything except your component